build/build.js

```diff
@@ -48,58 +29,53 @@ const bundleOptions = {
   banner,
   exports: 'named',
   format: 'umd',
-  moduleName,
+  name: moduleName,
 }
 
-function createBundle({ name, env, format }) {
-  return rollupBundle({ env })
-    .then(function (bundle) {
-      const options = Object.assign({}, bundleOptions)
-      if (format) options.format = format
-      const code = bundle.generate(options).code
-      if (/min$/.test(name)) {
-        const minified = uglify.minify(code, {
-          output: {
-            preamble: banner,
-            ascii_only: true, // eslint-disable-line camelcase
-          },
-        }).code
-        return write(`dist/${name}.js`, minified)
-      } else {
-        return write(`dist/${name}.js`, code)
-      }
+/**
+ * @param {String} name
+ * @param {String} [format='umd']
+ * @return {Promise<unknown>}
+ */
+async function createBundle(name, format) {
+  const bundle = await rollupBundle()
+  const options = { ...bundleOptions }
+
+  if (format) {
+    options.format = format
+  }
+
+  const { output } = await bundle.generate(options)
+  const code = output[0].code
+
+  if (/min$/.test(name)) {
+    const minified = uglify.minify(code, {
+      output: {
+        preamble: banner,
+        ascii_only: true, // eslint-disable-line camelcase
+      },
     })
-    .catch(logError)
+
+    if (minified.error) {
+      throw new Error(minified.error)
+    }
+
+    return write(`dist/${name}.js`, minified.code)
+  }
+
+  return write(`dist/${name}.js`, code)
 }
 
 // Browser bundle (can be used with script)
-createBundle({
-  name,
-  env: {
-    'process.env.NODE_ENV': '"development"',
```

Comment:
This was only used to generate a `minExt` in the build context from what I could tell, and that was not even referenced anywhere, so I removed this environment variable stuff.